product_route_profile/models/product_template.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2022 Akretion (https://www.akretion.com).
+# @author Kévin Roche <kevin.roche@akretion.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    route_profile_id = fields.Many2one("route.profile", string="Route Profile")
+    force_route_profile_id = fields.Many2one(
+        "route.profile",
+        string="Priority Route Profile",
+        company_dependent=True,
+        help="If defined, the "
+        "priority route profile will be used and will replace the "
+        "route profile, only for this company.",
+    )
+
+    route_ids = fields.Many2many(
+        compute="_compute_route_ids",
```

Comment:
Maybe a search method should be a great improvement

Reply:
True, I add it to the roadmap